### PR TITLE
fix: some language changes and minor typo fixes

### PR DESCRIPTION
**Pull request by astridlyre**
I just made a few small changes to the docs, to simplify some of the language and clean it up a little bit.

### CODE REVIEW

File: docs/introduction.mdx [hunk 1]
```diff
@@ -84,9 +84,13 @@ const sum = atom((get) => atoms.map(get).reduce((acc, count) => acc + count))
 ```
 
 ### Derived async atoms
-<img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
 
-You can make the read function an async function too.
+<img
+  src="https://img.shields.io/badge/-needs_suspense-black"
+  alt="needs suspense"
+/>
```
Comment (sandren) [inline docs/introduction.mdx:91]: The image tag must be in one line to render properly.

**Comment by sandren** [comment]
Thank you for all these fixes. Very much appreciated!

Just one issue with the badge not rendering. 🙂

File: docs/introduction.mdx (hunk 1 above)
Comment (astridlyre) [reply docs/introduction.mdx]: Whoops! I think that must have been prettier. Thanks. I've fixed it :smile:

File: docs/introduction.mdx [hunk 2]
```diff
@@ -84,9 +84,10 @@ const sum = atom((get) => atoms.map(get).reduce((acc, count) => acc + count))
 ```
 
 ### Derived async atoms
+
 <img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
```
Comment (dai-shi) [inline docs/introduction.mdx:88]: I guess we want the badge in the header. Does this work?
```suggestion
### Derived async atoms <img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
```

Comment (dai-shi) [reply docs/introduction.mdx]: Hmm, it doesn't work anyway. Let's leave it for now.
image

**Review by dai-shi — APPROVED** [review]
Wow, thanks so much for the fixes!

**Pull request merged**